pkg/capabilities/capabilities.go

```diff
@@ -0,0 +1,28 @@
+package capabilities
+
+import (
+	"fmt"
+	"github.com/syndtr/gocapability/capability"
```

Comment:
Out of scope of this PR but I was wondering if we are okey with gocapability BSD 2-Clause "Simplified" License. (I assume that the fact it's not very actively maintained is because is feature complete?)